examples/asyncapi-from-parser/index.spec.ts

```diff
@@ -0,0 +1,15 @@
+import {generate} from './index';
+
+describe('Should be able to process AsyncAPI object from parser', () => {
+  afterAll(() => {
+    jest.clearAllMocks();
+  });
+  test('and should log expected output to console', async () => {
+    const callbackMock = jest.fn();
+    await generate(callbackMock);
+    const expectedConsoleLog = `export interface AnonymousSchema_1 {
```

Comment:
I guess you're doing it this way because of how console.log is actually working. If example would write to file, and test would read the file and forward do snapshot testing, do you think it would be too much?

Reply:
I think we can use snapshots 🤔 Let me update the examples.